generators/entity-server/files.js

```diff
@@ -239,6 +239,37 @@ const serverFiles = {
                     renameTo: generator => `gatling/user-files/simulations/${generator.entityClass}GatlingTest.scala`
                 }
             ]
+        },
+        {
+            path: SERVER_TEST_SRC_DIR,
+            templates: [
+                {
+                    file: 'package/domain/EntityTest.java',
+                    renameTo: generator => `${generator.packageFolder}/domain/${generator.asEntity(generator.entityClass)}Test.java`
```

Comment:
```suggestion
                    renameTo: generator => `${generator.packageFolder}/domain/${generator.entityClass}Test.java`
```

Reply:
Thank you for the remark, I took it in account  in my last commit.

I fixed the missing import to UUID in MapperTest